pages/learn/electives.tsx

```diff
@@ -97,10 +97,9 @@ const LearnElectivesPage: CustomPage<{
                   electiveSubject={electiveSubject}
                   selected={selectedID === electiveSubject.id}
                   enrolled={enrolledElective?.id === electiveSubject.id}
-                  previouslyEnrolled={
-                    // Change to actual check when API's available
-                    electiveSubject?.id == previouslyEnrolled[0]
-                  }
+                  previouslyEnrolled={previouslyEnrolled.includes(
+                    electiveSubject?.id,
+                  )}
```

Comment:
There is an array `previouslyEnrolled` passed into the page and a boolean `previouslyEnrolled` as a prop of Elective List Item.

Reusing the same name for different variables can be confusing; I recommend `previouslyEnrolledIDs` and `isPreviouslyEnrolled`.